build/storage/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.qcow2
+*.tar.gz
+
+*.swp
+
```

Comment:
Don't know if we have a policy for this kind of files but I prefer not to include them in the repo...

Reply:
Hi Sander,
Thank you for the feedback. Can you explain in details your opinion? From my point of view `.gitignore` adds some value since it prevents from accidentally adding ide files and others.
`.tar.gz` file from this ` .gitignore` is leftover and not needed here and I've removed it. However, we can keep the rest.

Reply:
Yes, I'd also like to understand this, as I think having these in .gitignore is fine.

Reply:
The problem I have seen is that there are a lot of ide's and it is impossible to add all of those config files to the gitignore file. Also in most projects I worked in the maintainers didn't want to have them included (got even a stern warning once 😮), so I'm opinionated through others 🤦‍♂️. But as said, if we are ok with it, just add!👍